fastify.js

```diff
@@ -130,10 +129,12 @@ function fastify (options) {
   options.disableRequestLogging = disableRequestLogging
   options.ajv = ajvOptions
   options.clientErrorHandler = options.clientErrorHandler || defaultClientErrorHandler
-  options.exposeHeadRoutes = exposeHeadRoutes
 
   const initialConfig = getSecuredInitialConfig(options)
 
+  // TODO remove this, it should not be needed
```

Comment:
Does todo means that PR is not yet ready?

Reply:
no, I'll remove the todo.